### ISSUE TITLE
Make "Wanderers: Sestor: Bomb Zenith 1" choice less hidden?

### ISSUE THREAD

**Issue by Platypants** [endless-sky/endless-sky#3467]
On the "Bomb Zenith 1" mission, I saw nothing in the dialogue with Danforth that tells the player they may have a choice not to use the bomb device (meaning, to clear the Sestor at Farpoint and activate "Wanderers: Sestor Alt: Alnilam 1" instead). I only found out about this choice by looking at the source code later. I think there should be at least a hint to the player; perhaps a pop-up dialogue box can trigger after you take off from Farpoint. I see this as one of the bigger moral choices of the current storyline, but the player has no motivation to stay and fight the Sestor at Farpoint if they take Danforth at his word.

**Comment by tehhowch** [comment]
In 0.9.9 a bug which prevented dialogs from appearing when an NPC is successfully evaded is fixed. With this bug fixed, the player will indeed be notified of the alternative, should they choose to fight the Sestor and successfully disable them.
https://github.com/endless-sky/endless-sky/pull/2859

**Comment by Pointedstick** [comment]
Wow, I didn't even know this was an option.

**Comment by Amazinite** [comment]
Maybe some sort of on enter dialog that hints at destroying the fleet?
```
"Wanderers: Sestor: Bomb Zenith 1"
     on enter
          dialog `Danforth wasn't joking when he said that the device may be our only chance. There are almost a dozen Sestor capital ships, and countless smaller ships tearing the Navy to shreds. Even the Quarg warships are having trouble with an opposing fleet this size. Perhaps a more powerful fleet could defeat the automata, but it would be a difficult battle.`
```

**Comment by Brick63** [comment]
well you learn something new every day

**Comment by tehhowch** [comment]
Addressed in #3596
